jujugui/static/gui/src/app/test-init.js

```diff
@@ -66,15 +65,6 @@ describe('init', () => {
       container.getAttribute('class').should.include('container');
     });
 
-    it('activates the listeners for keyboard events', () => {
```

Comment:
This test was just testing that the library was doing what it says it's supposed to do. It was also attaching click-outside events which were causing cascading test failures.